homeassistant/components/tplink/entity.py

```diff
@@ -18,13 +24,26 @@
 _P = ParamSpec("_P")
 
 
-def async_refresh_after(
+def async_exception_wrap_refresh_after(
```

Comment:
On naming, I'm wondering if it's worth renaming this at all instead of just updating the docstring to reflect the change? This is still an internal decorator, so I'd be fine to keep the existing name to keep the diffs/logs more readable.

Reply:
For me no need to rename

Reply:
Renamed back to the original